shared/common-adapters/plain-input.stories.js

```diff
@@ -17,6 +19,138 @@ const commonProps = {
   style: {borderWidth: 1, borderStyle: 'solid', borderColor: globalColors.black_10},
 }
 
+type TestInputState = {
+  value: string,
+}
+class TestInput extends React.Component<{}, TestInputState> {
+  state = {value: ''}
+  // prettier-ignore
+  _input = React.createRef<typeof PlainInput>()
+
+  _insertText = (t: string) => {
+    const input = this._input.current
+    if (input) {
+      const selection = input.getSelection()
+      if (selection) {
+        this.setState(
+          s => {
+            const value = s.value.substring(0, selection.start) + t + s.value.substring(selection.start)
```

Comment:
mind changing the last `selection.start` to `selection.end`? (to match the behavior of the uncontrolled version)